### DISCUSSION TITLE
Bug in documentation - naming conventions for collections of form

### DISCUSSION THREAD

**Discussion by Jakubsoad** [symfony/symfony#42999] [Q&A] [answered]
Hi, I believe I found a bug in documentation, which took few hours of my life, so I want others to avoid it :)
In this part:
https://symfony.com/doc/current/form/form_themes.html#fragment-naming-for-collections

In the example, there is the field in form type TaskManagerType named taskLists. Next, there is a name convention for overriding the default template for this field: in this example it is:
```
{% block _task_manager_task_lists_* %}
    {# the collection field of TaskManager #}
{% endblock %}
```

But in Symfony (at least in 5.2) the working example is something like this (with camelCase for the name of the field, not snake):

```
{% block _task_manager_taskLists_* %}
    {# the collection field of TaskManager #}
{% endblock %}
```
(I'm not sure is a good place for reporting a bug in documentation, please let me know)

**Comment by Nek-** [comment, marked as answer]
Hello, thanks for the report, but indeed it should be an issue on [symfony-docs repository](https://github.com/symfony/symfony-docs/issues).

**Reply by Jakubsoad** [reply]
Ok, I wrote in right place, link is here:
https://github.com/symfony/symfony-docs/issues/15804
